ecosystem/platform/server/config/routes.rb

```diff
@@ -22,7 +22,16 @@
       confirmations: 'users/confirmations'
     }
   }
+
+  # Administration
   ActiveAdmin.routes(self)
+  constraints(lambda { |request|
+    user = request.env['warden'].user
+    user.present? && user.respond_to?(:is_root?) && user.is_root?
```

Comment:
nit: I don't think you need the `.present?` here as nil won't respond to `is_root`

Reply:
Done.